pycbc/live/snr_optimizer.py

```diff
@@ -0,0 +1,340 @@
+import argparse, numpy
+import time
+import logging
+import types
+from scipy.optimize import differential_evolution, shgo
+from pycbc import (
+    DYN_RANGE_FAC, waveform
+)
+from pycbc.types import zeros
+import pycbc.waveform.bank
+from pycbc.filter import matched_filter_core
+import pycbc.conversions as cv
+
+try:
+    import pyswarms as ps
+except:
+    ps = None
+
+# Set a minimum mass for points tried in optimization allowing for
+# minimal slop relative to the lightest template
+MIN_CPT_MASS = 0.99
```

Comment:
I would not be super happy about this special variable being defined here _and_ in the executable - SSOT. If it's defined here it should be used via import (`snr_optimizer.MIN_CPT_MASS` I think) in the script

Reply:
The comment in `pycbc_optimize_snr` is helpful for a definition of `MIN_CPT_MASS` so the way I've done it isn't the prettiest, what do you think?

Reply:
well, comments are cheap .. it seems OK atm